test/test_internal.jl

```diff
@@ -45,5 +45,5 @@ end
 
 @testset "Test internal values" begin
     sys_rts = create_rts_system()
-    @test validate_uuids(sys)
+    @test validate_uuids(sys_rts)
```

Comment:
This is a good example of why global variables are bad, even in tests.  test_busnumberchecks.jl defines ```sys``` as a global and so that is why this test was passing.